R/rap.R

```diff
@@ -179,6 +179,15 @@ prepare_wap <- function(.tbl, .f, check = TRUE) {
 #'      n = integer() ~ nrow(x)
 #'   )
 #'
+#' # rap is especially useful for iterating
+#' # over multiple models
+#' starwars %>%
+#'   nest_by(gender) %>%
```

Comment:
`nest_by()` will likely become `group_nest()`. https://github.com/tidyverse/dplyr/pull/3964 

We plan to merge this PR soon, so we can either wait for it, or specify the remote for it

Reply:
✅ @DavisVaughan can you s/nest_by/group_nest/